proposals/OBv3p0/specification.md

```diff
@@ -0,0 +1,374 @@
+
+* _credentialSubject.resultDescription_: This property is taken from the CLR and may follow the same recommendations. As with evidence, this property should be included in the credentialSubject object.
+
+* _Revoked Credentials_: VCs have a [credentialStatus](https://www.w3.org/TR/vc-data-model/#status) object. The id property must be a URL and the type property describes how the status of the credential may be checked.
+
+* _Skill Assertion_: As depicted in the example below, this new type of Open Badge verifies a claim that a subject has attained a skill. In this instance, a credentialSubject.resultDescription would be required because it would reference the skill descriptor of the achieved skill. Evidence may be included to support the claim.
+
+* _Verifiable Presentations_: One or more VCs can be combined into a [verifiable presentation](https://www.w3.org/TR/vc-data-model/#presentations) which is digitally signed by the presenter (typically the subject). A verifiable presentation is presented at the request of the verifier.  
+
+### Changes to the Official Open Source Validator
+
+This proposal will incur modifications to the validator and considerations as to how the validator handles prior versions of Open Badges and VC Open Badges. The working group can discuss topics such as whether validators should remain separate and if validated VC Open Badges should return VCs as receipts upon validation.
+
+### Changes to the API
+
+As with the validator, the BadgeConnect API will require adjustments to accommodate property changes. Conceptually, we will need to explore the flow and use cases of Open Badges as VCs.
+
+### Considerations
+
+* Learners may still share their badges online as HTML as they have in every version of Open Badges. Open Badges as VCs will be presented in response to verifiers requesting them from learner wallets. In addition to using badges for online portfolios, badges as VCs can be exchanged and evaluated as data.
+
+* Issuers will need to adopt cryptographic signature strategies as recommended by the VC community.
+
+## Examples
+
+### Example of an Open Badge as a Verifiable Credential
+
+  <pre>
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/openbadges/v3"
+      ],
+      "type": [
+        "VerifiableCredential",
+        "Open Badge"
+      ],
+      "id": "https://website-that-supports-or-displays-the-badge.html",
+      "issuer": {
+        "id": "did:example:issuer",
+        "name": "An Example Issuer",
+        "image": "https://example.org/logo.png",
+        "url": "https://example.org",
+        "email": "contact@example.org"
+      },
+      "issuanceDate": "2010-01-01T19:23:24Z",
+      "credentialSubject": {
+        "id": "did:example:learner",
+        "hasCredential": {
```

Comment:
does this imply a one-to-one relationship between badge and credential? would we want this to be an array?

on the contrary, what about a badge that requires multiple credentials to be satisfied?

Reply:
Like, "credential", "badge" is also an overloaded term. Sometimes it refers to the achievement definition (a badge that many learners might earn) and sometimes it refers to the assertion (the badge that Alice earned). 

The power of Open Badges often comes from the fact that they are granular and composable claims, each about one badge that one learner has earned. Allowing this to be an array instead of forming the array at a higher level (like a CLR) would undercut that composability and make it only possible for a recipient to share a bundle of all their badges that were issued together, or nothing at all. It probably doesn't make sense at this point to change the scope of what an Open Badges Assertion can claim in terms of the badge earned when we already have a great complementary concept of the CLR that can do that work (as well as introducing the possibility of using Verifiable Presentations as a mechanism to compose multiple credentials into one package that is greater than the sum of its parts.

Reply:
> what about a badge that requires multiple credentials to be satisfied?

This use case is very important to me, we have software in production that does this, and we would love to better make some of this data interoperable at the machine-readable level. To me this sounds like a use case for "Pathways", but perhaps there are some components that could be considered in badges beyond the human-readable criteria structure that allows people to express this information today. 

IMS is about to kick off a task force on Learning Pathways as part of the CASE work, that is also intended to be compatible with Open Badges and CLR as these specs all evolve together. The pathways task force kickoff registration is available [here](https://imsglobal.zoom.us/meeting/register/tJAtc-GrqzsrGdU2s66zVR64Xb4jyR718BdB) as part of the [Tech Congress agenda](https://www.imsglobal.org/event/ims-technical-congress-2021) for August 4th. 


However we name these concepts going forward and charter specifications and workgroups to cover them, these ideas should all fit together neatly. Here's the IMS groups covering this type of scope:
- Skill and Competency Definitions (Competencies and Standards Exchange  - CASE)
- Credential definitions and Assertions (Open Badges)
- Transcripts and Bundles of Credentials (Comprehensive Learner Record - CLR)
- Learning Pathways, both prescriptive and descriptive (CASE Pathways Task Force)